README.md

```diff
@@ -56,6 +63,50 @@ This option allows you to disable it if you want:
 
 ```cordova run android ios -- --livereload --ghostMode=false```
```

Comment:
It might be worth specifically calling out the extra -- required on the command-line somewhere in this doc, as it is easy to overlook and the purpose is probably not obvious to the user.